apps/server/lib/lexical/server/code_intelligence/completion/translations/macro.ex

```diff
@@ -438,6 +438,58 @@ defmodule Lexical.Server.CodeIntelligence.Completion.Translations.Macro do
     |> builder.boost()
   end
 
+  def translate(%Candidate.Macro{name: "test", arity: 1} = macro, builder, env) do
+    label = "#{macro.name} (Define an empty test/1)"
+
+    snippet = """
+    test "${0:name}"
+    """
+
+    env
+    |> builder.snippet(snippet,
+      detail: "",
+      kind: :class,
+      label: label
+    )
+    |> builder.boost(3)
```

Comment:
the higher the boost, the more important the thing; I think this is backwards, as this is just the stub.

Reply:
I was thinking in terms of chronological order, stub coming first. But I think your suggestion is better.